examples/vue-kitchen-sink/src/stories/custom-decorators.stories.js

```diff
@@ -30,6 +30,11 @@ export const template = () => ({
   template: '<my-button>MyButton with template</my-button>',
 });
 
+export const withData = ({ parameters, ...rest }) =>
+  console.log(parameters) || {
+    template: `<pre>${JSON.stringify({ ...rest, parameters }, null, 2)}</pre>`,
+  };
```

Comment:
you dont' want the console to come after the other template string?  I'm not sure what the main goal of this function is trying to achieve via the naming of it.